domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/casefield/CaseFieldEntityACLValidatorImpl.java

```diff
@@ -14,12 +21,15 @@ public ValidationResult validate(final CaseFieldEntity caseField,
                                      final CaseFieldEntityValidationContext caseFieldEntityValidationContext) {
 
         final ValidationResult validationResult = new ValidationResult();
-
+        Set<String> caseRoles = caseFieldEntityValidationContext.getCaseRoles().stream().map(CaseRoleEntity::getReference).collect(Collectors.toSet());
```

Comment:
we could have this:
`.stream().map(CaseRoleEntity::getReference).collect(Collectors.toSet())` inside the `getCaseRoles()` method itself. Most of the times you'll only be interested on the reference